### PR TITLE
all errors which are reported by thrown canot be adjusted by user

### PR DESCRIPTION
**Pull request by OsmondJiang**
give us a clear way to determine which error can be adjusted by user. 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/4801)

### CODE REVIEW

File: docs/specs/toc.yml [hunk 1]
```diff
@@ -1399,3 +1399,30 @@ outputs:
...
   docs/f1/toc.json:
   docs/f2/toc.json:
+---
+# turn skip-level toc error to warning
+inputs:
+  docfx.yml: |
+    rules:
+      invalid-toc-level: warning
+  docs/TOC.md: |
+    # level 1
```
Comment (yufeih) [inline docs/specs/toc.yml]: can the first level be `##` ?

Comment (OsmondJiang) [reply docs/specs/toc.yml]: yes, check test `more than 6 leading count of #`

Comment (yufeih) [reply docs/specs/toc.yml]: Is it implemented like that or designed like that, @superyyrrzz what does v2 do here?

Comment (OsmondJiang) [reply docs/specs/toc.yml]: tested here: https://github.com/MicrosoftDocs/engineering/pull/514#issuecomment-505732272

Comment (OsmondJiang) [reply docs/specs/toc.yml]: supported in v2, built page: https://review.docs.microsoft.com/en-us/engineering/projects/toc.json?branch=pr-en-us-514

**Pull request merged**